File: hotel_search/v2/by_offer/node/hotel_search.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// Confirm the availability of a specific offer
amadeus.shopping.hotelOffer('XXX').get()
  .then(function (response) {
    console.log(response);
  }).catch(function (response) {
    console.error(response);
  });

File: hotel_search/v2/by_offer/python/hotel_search.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client()

try:
    # Confirm the availability of a specific offer
    offer_availability = amadeus.shopping.hotel_offer('F5B20A8E3C7EED38BDB931B587B3435232A8C804E3271D96F9B5826C326F61FA').get()
    print(offer_availability.data)
except ResponseError as error:
    raise error
